COVIDWatch iOS/ViewControllers/Test.swift

```diff
@@ -333,14 +280,13 @@ class Test: UIViewController {
                             self!.agree.center.y = self!.agree.center.y - 950
                             self!.disagree.center.y = self!.disagree.center.y - 950
                             
-                            self!.agreeLbl.center.y = self!.agreeLbl.center.y - 950
-                            self!.disagreeLbl.center.y = self!.disagreeLbl.center.y - 950
+                            self!.agree.text.center.y = self!.agree.center.y
```

Comment:
If would be great if you could modify this to use `if let` syntax instead of force unwrapping.  This could cause crashes.

Reply:
Great idea!  If someone else hasn't implemented this I can definitely do so.